src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -126,12 +126,17 @@ LandmarkEfficientOptimalSharedCostAssignment::LandmarkEfficientOptimalSharedCost
     const LandmarkGraph &graph,
     lp::LPSolverType solver_type)
     : LandmarkCostAssignment(operator_costs, graph),
-      lp_solver(solver_type) {
+      lp_solver(solver_type),
+      lp(initial_lp()) {}
```

Comment:
I think we usually add a line break after `{` in situations like this. Also, names of functions should be verbs, so I'd change this to
```
      lp(build_initial_lp()) {
}
```